doc/getting_started/how_to_run_training.rst

```diff
@@ -47,6 +47,11 @@ instantiating the method you selected. See :doc:`supported_algorithms` for more
 
 
 
+..  note::
+    If you have defined the ``optimizer_fn`` function in your Renate config, do not pass values for the keys
```

Comment:
Is there a check for this in the code?

Reply:
Yes. `run_training_job.py` will not add these params in the `argparse` code and therefore the script will fail if these arguments are passed but not expected.